client/my-sites/site-settings/redirect-to/README.md

```diff
@@ -0,0 +1,19 @@
+Check and restrict site access rights to the disconnection process.  
+==================================================================
+
+This component redirects to ```/settings/general``` for non-Jetpack
+and Atomic sites. The redirection happens automatically prior to rendering
+the site as checks are placed in lifecycle calls.
+
+#### TODO
+- possibly find better naming for the function ```verifySiteIsJetpack```
+since we check BOTH whether a given site is a Jetpack site AND whether
+it is non-Atomic. In the latter case, we want to restrict access
+to the (dis)connection options.
+For more context, see discussion at https://github.com/Automattic/wp-calypso/pull/17555
+
+- generalize the redirection url. Currently, we redirect
```

Comment:
Leaving this to your personal preference, but I don't think it adds a lot of burden to have this one implemented in the first version of the HoC with a prop with a default value, that would allow easy overriding if necessary.

Reply:
Included 👍